--- apertium-oc-ca-1.0.5/Makefile.am.orig	2008-07-21 13:02:56.000000000 +0200
+++ apertium-oc-ca-1.0.5/Makefile.am	2011-01-02 09:39:33.785898177 +0100
@@ -267,9 +267,9 @@
 
 install-data-local:
 	apertium-gen-modes modes.xml apertium-$(PREFIX1)
-	$(INSTALL_DATA) oc-ca.mode $(apertium_oc_camodesdir)
-	$(INSTALL_DATA) ca-oc.mode $(apertium_oc_camodesdir)
-	$(INSTALL_DATA) oc_aran-ca.mode $(apertium_oc_camodesdir)
-	$(INSTALL_DATA) ca-oc_aran.mode $(apertium_oc_camodesdir)
+	$(INSTALL_DATA) oc-ca.mode $(DESTDIR)$(apertium_oc_camodesdir)
+	$(INSTALL_DATA) ca-oc.mode $(DESTDIR)$(apertium_oc_camodesdir)
+	$(INSTALL_DATA) oc_aran-ca.mode $(DESTDIR)$(apertium_oc_camodesdir)
+	$(INSTALL_DATA) ca-oc_aran.mode $(DESTDIR)$(apertium_oc_camodesdir)
 	
 CLEANFILES = -r $(TARGETS)
