--- apertium-oc-ca-1.0.5/apertium-oc-ca.oc-ca.t1x.orig	2008-07-21 13:02:56.000000000 +0200
+++ apertium-oc-ca-1.0.5/apertium-oc-ca.oc-ca.t1x	2011-01-02 10:00:40.661909072 +0100
@@ -1283,6 +1283,7 @@
 			<choose><!-- aquests no són ref-->
 				<when>
 					<test>
+					    <and>
 						<equal caseless="yes">
 							<clip pos="2" side="sl" part="pers"/>
 							<lit-tag v="p3"/>
@@ -1291,6 +1292,7 @@
 							<clip pos="2" side="sl" part="nbr"/>
 							<lit-tag v="sg"/>
 						</equal>
+					    </and>
 					</test>
 					<choose>
 						<when>
--- apertium-oc-ca-1.0.5/apertium-oc-ca.ca-oc.t1x.orig	2008-07-21 13:02:56.000000000 +0200
+++ apertium-oc-ca-1.0.5/apertium-oc-ca.ca-oc.t1x	2011-01-02 10:21:06.057902367 +0100
@@ -988,6 +988,7 @@
 			<choose><!-- aquests no són ref-->
 				<when>
 					<test>
+					    <and>
 						<equal caseless="yes">
 							<clip pos="2" side="sl" part="pers"/>
 							<lit-tag v="p3"/>
@@ -996,6 +997,7 @@
 							<clip pos="2" side="sl" part="nbr"/>
 							<lit-tag v="sg"/>
 						</equal>
+					    </and>
 					</test>
 					<choose>
 						<when><!-- m'ha passat-> m'a arribat-->
@@ -1203,6 +1205,7 @@
 			<choose><!-- aquests no són ref-->
 				<when>
 					<test>
+					    <and>
 						<equal caseless="yes">
 							<clip pos="2" side="sl" part="pers"/>
 							<lit-tag v="p3"/>
@@ -1211,6 +1214,7 @@
 							<clip pos="2" side="sl" part="nbr"/>
 							<lit-tag v="sg"/>
 						</equal>
+					    </and>
 					</test>
 					<choose>
 						<when><!-- m'ha passat-> m'a arribat-->
